pkg/transformer/openshift/openshift.go

```diff
@@ -244,5 +246,59 @@ func (o *OpenShift) Deploy(komposeObject kobject.KomposeObject, opt kobject.Conv
 }
 
 func (o *OpenShift) Undeploy(komposeObject kobject.KomposeObject, opt kobject.ConvertOptions) error {
-	return errors.New("Not Implemented")
+	//Convert komposeObject
+	objects := o.Transform(komposeObject, opt)
+
+	oclient, err := o.getOpenShiftClient()
+	if err != nil {
+		return err
+	}
+	kclient, namespace, _ := o.Kubernetes.GetKubernetesClient()
```

Comment:
@surajssd I didn't notice that, thanks for the trace :+1: 
and yeah `o.GetKubernetesClient()`, it works, I have updated the PR.